kyuubi-common/src/test/java/org/apache/kyuubi/tags/DeltaTest.java

```diff
@@ -27,4 +27,4 @@
 @TagAnnotation
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.METHOD, ElementType.TYPE})
-public @interface DataLakeTest {}
+public @interface DeltaTest {}
```

Comment:
Not a big issue, but shall we add a parent tag for `DeltaTest` and `IcebergTest` ?

Reply:
I think we use tags here to mark tests which not support specific Spark versions, and due to the limitation of scalatest maven plugin, such as collection operation, the workflow scripts are not elegant now. Would you explain the benefit of adding a parent tag?

Reply:
Not a strong opinion. I'm fine with both.